### DISCUSSION TITLE
Why can't I get Rai Pal to find my Epic games library?

### DISCUSSION THREAD

**Discussion by hdmechanic33** [Raicuparta/rai-pal#157] [Q&A]
It's showing all of my Steam and GOG games, but all of my Epic games are missing!  What do I need to do?

**Comment by Leaderwookie** [comment]
Same problem here

**Comment by Raicuparta** [comment]
have you guys checked if the filters are set to show only VR games? Some other users had this problem. A recent Rai Pal update made the filters a bit more confusing, since picking "flat only" in the filters will now hide games where Rai Pal can't guess whether or not they're flat. While in previous versions, Rai Pal would just presume any other game was flat.

**Comment by Raicuparta** [comment]
Version 0.8.0 has a new filter system that hopefully helps with this. If it's still confusing, I'll go back to presuming a game is flat until proven otherwise.
